src/OSPSuite.InstallationValidator.Core/Domain/StartableProcess.cs

```diff
@@ -46,14 +46,15 @@ public void Dispose()
 
       public virtual void Wait(CancellationToken token)
```

Comment:
Here's from msdn

> true to reliably release the handle during the finalization phase; false to prevent reliable release (not recommended).

I'm not 100% sure, but reading between the lines here, this  means the handle won't be released when signaled. I think that's correct because the handle belongs to the process in this case. Most cases it does not. I assume why it's 'not recommended'